docs/lookout-tool.md

```diff
@@ -1,22 +1,22 @@
-# lookout-sdk Binary
+# lookout-tool Binary
 
 _For the **lookout-sdk** library to develop new analyzers go to [**lookout-sdk**](https://github.com/src-d/lookout-sdk) repository._
```

Comment:
Isn't this to be changed too?

Reply:
nope

Reply:
@se7entyse7en because that other points to **lookout-sdk** library, at https://github.com/src-d/lookout-sdk.
That's a complete different thing:
- ~lookout-sdk~ &rarr; `lookout-tool` : a CURL-like tool to call analyzers
- **lookout-sdk** : a library to use as a dependency when creating analyzers and storing protos 

We decided to rename the binary to avoid that confusion ;)